pip/worker/pypi.go

```diff
@@ -4,17 +4,18 @@ package worker
 
 import (
 	"encoding/json"
-	"errors"
 	"io"
-	"net/http"
 	"reflect"
 	"strings"
 
+	"github.com/opensbom-generator/parsers/internal/helper"
 	"github.com/opensbom-generator/parsers/meta"
+	"github.com/pkg/errors"
 )
 
-var (
-	errorPypiCouldNotFetchPkgData = errors.New("could not fetch package data from PyPI")
+const (
+	pypiBaseURL                 = "https://pypi.org"
```

Comment:
Sometimes, organizations mirror pypi.org internally (for example: https://www.jfrog.com/confluence/display/JFROG/PyPI+Repositories). Perhaps there is a way to configure this in a file and read that information in?